src/components/setup/DataElement.jsx

```diff
@@ -27,7 +28,9 @@ const DataElement = ({
             </tr>
             <tr>
                 <th>{i18n.t('Description')}</th>
-                <td>{description}</td>
+                <td>
+                    {description} {i18n.t('Data source')}: {source}
+                </td>
```

Comment:
Data source feels important to me and probably making it a row on its own provides more emphasis.

Reply:
Fixed.